### ISSUE TITLE
RNAsubopt generates invalid structures

### ISSUE THREAD

**Issue by reisalex** [ViennaRNA/ViennaRNA#88]
Please see the following RNAsubopt result (ViennaRNA v2.4.14). There are both invalid base pairs and unbalanced parentheses. Of note, this issue did not manifest in an older version (v2.4.10).
```
$ RNAsubopt
Use '&' to connect 2 sequences that shall form a complex.

Input string (upper or lower case); @ to quit
....,....1....,....2....,....3....,....4....,....5....,....6....,....7....,....8
CCCUUGCGGCGG&ACCUCCUUA
length = 21
CCCUUGCGGCGG&ACCUCCUUA  -4.40   1.00
.......((.()&.))(()...  -4.10
.......((.()&.))())...  -4.20
.......((.()&.)).()...  -4.30
.......((.((&.)).))...  -4.40
```

**Comment by RaumZeit** [comment]
Thank you for the bug report!
I've isolated the problem and already fixed the bug in our development branch. The new release `2.4.15` of the `ViennaRNA Package` that includes the fix is on it's way and will be released within the next few days...

Sorry for the inconvenience!
Best,
Ronny

**Comment by RaumZeit** [comment]
Fixed with version `2.4.15`
